modules/install-dnsmasq/README.md

```diff
@@ -50,6 +50,8 @@ The `install-dnsmasq` script accepts the following arguments:
 * `consul-ip IP`: The IP address to use for Consul. Optional. Default: `127.0.0.1`. This assumes a Consul agent is 
   running locally and connected to a Consul cluster.
 * `consul-dns-port PORT`: The port Consul uses for DNS requests. Optional. Default: `8600`.
+* `dnsmasq-listen-address IP`: The IP address for dnsmasq to listen on. Optional. Defaults to the value of `consul-ip`. 
+This assumes that the newtwork interface for the provided IP has been configured
```

Comment:
What does the last sentence mean?

Reply:
I left this to make sure the user knows that depending on the address they choose, they may need to set up the network interface at boot time prior to starting dnsmasq. An example use case is that if you're configuring dnsmasq to bind to a dummy interface. This could be more specific to my use-case than I realize, so I'm happy to remove it. I mostly didn't want users to think that they could simply put any IP address as a parameter of this script and that everything will work without extra configuration. Admittedly I'm no expert in networking on Linux machines, so I definitely could have some poorly chosen language here and am not opposed to doing more research and rephrasing if you think it's valuable

Reply:
Gotcha. Perhaps rephrasing to something like, "Make sure that the network interface you provide for the IP has already been configured before you pass it to dnsmasq" would be a bit clearer?

Reply:
Agreed!